netbox/templates/base.html

```diff
@@ -67,7 +67,7 @@ <h4><i class="mdi mdi-alert"></i> Maintenance Mode</h4>
                     <p class="text-muted">{{ settings.HOSTNAME }} (v{{ settings.VERSION }})</p>
                 </div>
                 <div class="col-xs-4 text-center">
-                    <p class="text-muted">{% now 'Y-m-d H:i:s T' %}</p>
+                    <p class="text-muted">{% now_span %} {% now 'T' %}</p>
```

Comment:
The template tag should be able to render the entire date string.

Reply:
The problem with the old code was that the date format was hard coded. I decided to use two template tags to respect the configured date format while preserving the time zone which is not intended to be part of the configurable date format strings. What do you have in mind? Do you want to keep the hard coded format?